### DISCUSSION TITLE
CFG file for Training a custom NER in conjunction with  web_trf  transformer Spacy v3

### DISCUSSION THREAD

**Discussion by badri-rutgers** [explosion/spaCy#10062] [Help: Model Advice] [answered]
can any one provide a working CFG file for spacyv3 for training a NER for one custom tag. I have used the one shown below, but I get all scores for model as 0  for each epoch .

components.transformer]
source = "en_core_web_trf"

[components.ner]
factory = "ner"
incorrect_spans_key = null
moves = null
scorer = {"@scorers":"spacy.ner_scorer.v1"}
update_with_oracle_cut_size = 100

[components.ner.model]
@architectures = "spacy.TransitionBasedParser.v2"
state_type = "ner"
extra_state_tokens = false
hidden_width = 128
maxout_pieces = 3
use_upper = false
nO = null

[components.ner.model.tok2vec]
@architectures = "spacy-transformers.TransformerListener.v1"
grad_factor = 1.0
pooling = {"@layers":"reduce_mean.v1"}
upstream = "transformer"

[components.tagger]
source = "en_core_web_trf"
replace_listeners = ["model.tok2vec"]

[components.parser]
source = "en_core_web_trf"
replace_listeners = ["model.tok2vec"]

[components.tok2vec]
source = "en_core_web_trf"

**Comment by polm** [comment, marked as answer]
Closing as duplicate of #10064
